cypress/support/commands.js

```diff
@@ -109,8 +109,8 @@ Cypress.Commands.add('createIndex', (index, settings = {}) => {
   cy.request('PUT', `${Cypress.env('opensearch')}/${index}`, settings);
 });
 
-Cypress.Commands.add('ingestDocument', (indexId, documentJSON) => {
-  cy.request('POST', `${Cypress.env('opensearch')}/${indexId}/_doc`, documentJSON);
+Cypress.Commands.add('ingestDocument', (indexId, sample_document) => {
```

Comment:
The variable name was changed by mistake here I think. We should leave this as `documentJSON` to make the command's functionality a little more clear.

Reply:
ah yes

Reply:
fixed